src/services/database.ts

```diff
@@ -93,3 +102,28 @@ const CriteriaIndexMap: Record<Exclude<SearchCriteria, SearchCriteria.ALL>, Tabl
   vin: 'VinIndex',
   trailerId: 'TrailerIdIndex',
 };
+export const postTechRecord = async (request: any) => {
```

Comment:
```suggestion
export const postTechRecord = async (request: any) => Promise<reponse | null>{
```
Would be nice to have a return type on the function